recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "0.2.4" %}
+{% set version = "0.3.0" %}
 
 package:
    name: intake-xarray
    version: {{ version }}
 
 source:
-  url: https://github.com/ContinuumIO/intake-xarray/archive/{{ version }}.tar.gz
-  sha256: d64c61040ab02b19e3c1c2e329cd950074a2624091d21945d672612fa090b438
+  url: https://pypi.io/packages/source/i/intake-xarray/intake-xarray-{{ version }}.tar.gz
```

Comment:
We probably cannot run the full test suite with the PyPI source. (I'm fine dropping the tests BTW.)

Reply:
Yeah I don't know why they are failing though

Reply:
My guess is missing files that are not part of the source distribution.

Reply:
yeah that's what @martindurant said too